apparmor.d/groups/apt/dpkg-preconfigure

```diff
@@ -72,9 +84,12 @@ profile dpkg-preconfigure @{exec_path} {
   owner /var/cache/debconf/tmp.ci/*.template.@{rand6} w,
   owner /var/cache/dictionaries-common/flag-wordlist-new w,
   owner /var/log/unattended-upgrades/unattended-upgrades-dpkg.log rw,
+  /var/cache/debconf/{config,passwords,templates}.dat{,-old,-new} r,
+  /var/cache/debconf/tmp.ci/ rw,
 
   owner @{tmp}/*.template.* rw,
   owner @{tmp}/*.config.* rwPUx,
+  owner @{tmp}/** mr,
```

Comment:
Memory mapping without execution. Is it a failed `dpkg-preconfigure
` run or just notice?